trace.go

```diff
@@ -49,6 +49,10 @@ const (
 	TraceBatchReplication
 	// TraceBatchKeyRotation will trace batch keyrotation operations.
 	TraceBatchKeyRotation
+	// TraceBatchExpire will trace batch expiration operations.
+	TraceBatchExpire
+	// TraceBatch will trace all batch operations.
+	TraceBatch
```

Comment:
We should define `TraceBatch` outside of this `iota` based expressions. 

`TraceBatch` is simply,
```go
const(
  TraceBatch = TraceBatchReplication|TraceBatchKeyRotation|TraceBatchExpire // |TraceBatch<NextFeature>
)
```